cli/command/load_balancer_command.go

```diff
@@ -77,6 +77,11 @@ func (l *LoadBalancerCommand) GetCommand() cli.Command {
 						Value: 2,
 						Usage: "number of consecutive failures required to count as unhealthy",
 					},
+					cli.IntFlag{
+						Name:  "idle-timeout",
+						Value: 60,
+						Usage: "idle timeout of the load balancer in seconds",
```

Comment:
I think we should give users some more help on what this value is and how rare it would be to set it to anything other than default? 

I've personally never set this value to anything other than default...

Reply:
Should that explanation be here or in the documentation? It seems like I should put a more verbose description in the documentation rather than here in the CLI context.

Reply:
I've opted to add a couple of sections to the CLI and Terraform Plugin reference pages with a more thorough explanation of what the idle timeout is.